cmake/FindOpenVDB.cmake

```diff
@@ -619,10 +619,16 @@ if(OpenVDB_USES_LOG4CPLUS)
 endif()
 
 if(OpenVDB_USES_IMATH_HALF)
-  find_package(IlmBase REQUIRED COMPONENTS Half)
+  find_package(Imath CONFIG)
+  if (NOT TARGET Imath::Imath)
+    find_package(IlmBase REQUIRED COMPONENTS Half)
+  endif()
+
   if(WIN32)
     # @note OPENVDB_OPENEXR_STATICLIB is old functionality and should be removed
-    if(OPENEXR_USE_STATIC_LIBS OR (${ILMBASE_LIB_TYPE} STREQUAL STATIC_LIBRARY))
+    if(OPENEXR_USE_STATIC_LIBS OR
+        (${ILMBASE_LIB_TYPE} STREQUAL STATIC_LIBRARY)
+        (${IMATH_LIB_TYPE} STREQUAL STATIC_LIBRARY))
```

Comment:
Is there some kind of implicit `OR` between these two parenthesised statements? Looks a bit odd

Reply:
definitely need an OR in there.

Reply:
Typo.